package.json

```diff
@@ -35,7 +35,7 @@
     "react-static-container": "^1.0.0-alpha.1"
   },
   "peerDependencies": {
-    "react-relay": "0.7.1"
+    "react-relay": "^0.7.1"
```

Comment:
*isomorphic-relay* depends heavily on Relay internal API, that's why it uses exact version. But it's probably safe to write `"0.7.1 || 0.7.2"`. Have you tested it with 0.7.2?